zebra-chain/src/transaction/arbitrary.rs

```diff
@@ -104,6 +104,26 @@ impl Transaction {
             )
             .boxed()
     }
+    /// Generate a proptest strategy for V5 Transactions
+    pub fn v5_strategy(ledger_state: LedgerState) -> BoxedStrategy<Self> {
+        (
+            transparent::Input::vec_strategy(ledger_state, 10),
+            vec(any::<transparent::Output>(), 0..10),
+            any::<LockTime>(),
+            any::<block::Height>(),
+            any::<Vec<u8>>(),
+        )
+            .prop_map(
+                |(inputs, outputs, lock_time, expiry_height, rest)| Transaction::V5 {
+                    inputs,
+                    outputs,
+                    lock_time,
+                    expiry_height,
+                    rest,
```

Comment:
The field order also needs to be updated here - the current field order will trigger a clippy lint on recent nightlies.

Reply:
Sorry, missed that one. Fixed in https://github.com/ZcashFoundation/zebra/pull/1824/commits/21b85a33f715408c04f8c4b314516d10fa1e7b6f